### PR TITLE
Fixes metric grouping for CPU usage graphs.

### PR DESCRIPTION
**Pull request by tmacam**
# Description

"Total CPU Usage" graph is displaying `container_cpu_usage_seconds_total` but it is not grouping it by application and is instead using the `pod` field as its discriminating field.

Each application is re-deployed daily and receives a new pod-id. As this metric graph uses the `pod` id in its legend, the same application ends up in represented as a series of disconnected metrics.

This PR fixes the metric to grouping distinct "pods" under the same by "application id" using some metric-math. This in turn will allow us to observe how a given application behaves over time.

## Reference

Refence links for this PR:

* https://grafana.com/docs/loki/latest/logql/#label_replace
* https://stackoverflow.com/questions/66669688/group-labels-in-a-prometheus-query-based-on-regex
* https://stackoverflow.com/questions/49334224/prometheus-group-by-substring-of-label

## Issue reference

Please reference the issue this PR will close: #5524 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: grafana/grafana-system-services-dashboard.json [hunk 1]
```diff
@@ -154,9 +154,9 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(container_cpu_usage_seconds_total{pod=~\"(dapr-sentry|dapr-sidecar-injector|dapr-placement-server|dapr-operator).*\"}[5m])) by (pod)",
+          "expr": "sum(label_replace(rate(container_cpu_usage_seconds_total{pod=~\"(feed-generator|hashtag-actor|hashtag-counter|pubsub-workflow|validation-worker|message-analyzer).*\", namespace=\"longhaul-test\"}[5m]), \"app_name\", \"$1\", \"pod\", \"(.*-app)-.*\")) by (app_name)",
```
Comment (daixiang0) [inline grafana/grafana-system-services-dashboard.json:157]: Where are those pods from?

Comment (tmacam) [reply grafana/grafana-system-services-dashboard.json:157]: Those come from the metrics we are obtaining from k8s itself as I understand.

**Comment by daixiang0** [comment]
Could you share the graph before and after? That would be nice.

**Review by artursouza — APPROVED** [review]

**Pull request merged**